docs/manuals/userman/configuration/config-checker.md

```diff
@@ -24,18 +27,29 @@ Here is the list of the possible issues that the checker could return:
 - Depending on the `storage_type` value of a `DataNodeConfig^`, an `ERROR` issue is created if a specific required
   property is missing.
 - An `ERROR` issue is created if one of the `inputs` and `outputs` parameters of a `TaskConfig^` does not correspond
-  to a `DataNodeConfig`.
+  to a `DataNodeConfig` or a list of `DataNodeConfig^`.
 - A `WARNING` issue is created if a `TaskConfig^` has no input and no output.
+- An `ERROR` issue is created if any `DataNodeConfig^` of the `inputs` or `outputs` has the same name as one of the
+  attributes or properties of the `TaskConfig^`.
 - An `ERROR` issue is created if the `function` parameter of a `TaskConfig^` is not a callable function.
 - An `ERROR` issue is created if one of the tasks provided in the `sequences` parameters does not does not exist in
   the corresponding list of tasks in `tasks` parameter of the `ScenarioConfig^`.
-- An `ERROR` issue is created if a `ScenarioConfig^` has no task configuration defined.
-- A `WARNING` issue is created if a `ScenarioConfig^` has no additional data node configuration defined.
+- An `ERROR` issue is created if a `ScenarioConfig^` has no task configuration defined or does not correspond to a
+  `TaskConfig` or a list of `TaskConfig^`.
+- A `ERROR` issue is created if a `ScenarioConfig^` has no additional data node configuration defined or an additional
+  does not correspond to a `DataNodeConfig` or a list of `DataNodeConfig^`.
```

Comment:
```suggestion
- A `ERROR` issue is created if any additional data node configuration defined
  does not correspond to a `DataNodeConfig^`.
```

Reply:
We need to specify "any additional data node" of a ScenarioConfig. Without the ScenarioConfig, the sentence doesn't make sense to me